### DISCUSSION TITLE
Codespace stuck on "Setting up your codespace"

### DISCUSSION THREAD

**Discussion by dineshh-choudhary** [community/community#58548] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

After clicking on codespace from GitHub, it is not opening codespace and the screen is stuck forever on "Setting up your codespace". Facing it from last 2 days, earlier it was working just fine.
<img width="966" alt="Screenshot 2023-06-20 at 5 14 27 PM" src="https://github.com/community/community/assets/75560994/396ce767-aff8-439f-820c-8aacec9f9a3c">
<img width="428" alt="Screenshot 2023-06-20 at 5 14 21 PM" src="https://github.com/community/community/assets/75560994/e7d142c9-a808-4319-a70c-180c3887e23d">

**Comment by bdmac** [comment]
👋 Hi! I'd love to look into this for you but it seems you've deleted your codespace already. Is this still happening? What repository are you seeing this on?

**Reply by DineshhChoudhary** [reply]
Hi @bdmac Yes, its still happening!! I am not able to access codespace, from what I debugged till now its happening because of wifi, I am able to connect to codespace via mobile hotspot. But interestingly there is no firewall setting which block this access, as on the same wifi I was able to access 2 days back.

**Reply by Hisaalu** [reply]
For my case i duo boot, so when i switch from ubuntu "firefox but running in google chrome" to windows, it successfully opens "code space" on chrome for windows

**Comment by Hemant-HJ** [comment]
I am also stuck at the same page.
It happened to me when I stopped the codepsace manually and restarted it.

**Comment by lumartins311** [comment]
Pudieron solucionarlo? me está pasando lo mismo!!!

**Reply by Hemant-HJ** [reply]
yeah try dowloading touchvpn extension for chrome it works fine with it I guess its something related to network issue.
It works fine it i turn any vpn on or change my network.

**Reply by lumartins311** [reply]
gracias!

**Reply by romi615** [reply]
It doesn't work, I tried. Do you have any other solution I'm stucked "setting up your codespace".

**Reply by sanchalee19** [reply]
I switched from home wifi to mobile hotspot after that it started working. Even after switching back to home wifi it worked.

**Comment by Magnuaf** [comment]
Hi! I have the same issue even when using a VPN. What can I do?

**Comment by rohantalwadia** [comment]
![image](https://github.com/community/community/assets/64331914/85414d4f-aaa1-4820-9772-8b92c2cb6987)

Facing a similar issue, stuck here.

**Reply by Magnuaf** [reply]
I am using VS code local on my mac and I am not facing any issues anymore.

**Reply by romi615** [reply]
I'm also facing same issue, can anyone tell how to resolve this problem ?

**Reply by abisarwan-go** [reply]
have you tried to open it in chrome or another browser ?

**Reply by romi615** [reply]
Yes, I tried but nothing happened, And I also tried Touch VPN but this way also didn't work.

**Reply by sandesh2526** [reply]
It worked for me with the *TouchVPN*. At start the *TouchVPN* was connected to the *UK* and it was showing the same error but by changing it to *USA* I was finally able to launch the Codespace.

**Reply by Sudarshanhalse1** [reply]
> ![image](https://private-user-images.githubusercontent.com/64331914/260297034-85414d4f-aaa1-4820-9772-8b92c2cb6987.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2ODg5MTYsIm5iZiI6MTcxODY4ODYxNiwicGF0aCI6Ii82NDMzMTkxNC8yNjAyOTcwMzQtODU0MTRkNGYtYWFhMS00ODIwLTk3NzItOGI5MmMyY2I2OTg3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDA1MzAxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5MWE2NjgwZjdmNTY2YmYwMjhhZGJiYTk0MTljMTFlNDhlZmM4NTYxOWYwMGU0MzM4MDJhZmY3MDNjNjIxNTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zDoxhKOzmwDBdT75BSHfkowvSGBkFccW_L7MywDMkNk)
> 
> Facing a similar issue, stuck here.

How to fix it

**Reply by iam-tusharsinghbisht** [reply]
> > ![image](https://private-user-images.githubusercontent.com/64331914/260297034-85414d4f-aaa1-4820-9772-8b92c2cb6987.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2ODg5MTYsIm5iZiI6MTcxODY4ODYxNiwicGF0aCI6Ii82NDMzMTkxNC8yNjAyOTcwMzQtODU0MTRkNGYtYWFhMS00ODIwLTk3NzItOGI5MmMyY2I2OTg3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDA1MzAxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5MWE2NjgwZjdmNTY2YmYwMjhhZGJiYTk0MTljMTFlNDhlZmM4NTYxOWYwMGU0MzM4MDJhZmY3MDNjNjIxNTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zDoxhKOzmwDBdT75BSHfkowvSGBkFccW_L7MywDMkNk)
> > Facing a similar issue, stuck here.
> 
> How to fix it

Try once with your mobile hotspot once its setup you can change it to your wifi.

**Reply by sanchalee19** [reply]
Try changing your wifi networks like switching to mobile hotspot... It worked for me

**Comment by Alexandar016** [comment]
I have the same issue. I was using codespaces for my github repo. Now i can not even open vscode.dev and also i always get stuck in "we are setting up your codespace" window. Please help
P.S. This is only happening on my Ipad. On other platforms and systems it runs, even with same internet connection.

***** NEW UPDATE *****
I fix the issue by updating the IOS of Ipad. Apparently there was an update, which i missed and that solved the bug with codespaces as well with vscode.dev.
So just check if all your apps and IOS's are up to date. It worked for me.

**Reply by grace000** [reply]
This worked for me, thanks!!

**Reply by romi615** [reply]
Can you tell me how to fix "setting up your codespace" this problem ? And I also tried such as 1 ) Opened it in another browser. 2 ) Downloaded Touch VPN also. 3 ) I have checked my all Apps and OS to update but no update is available. So guys, do you have another solution ?

**Reply by ichnoguy** [reply]
This worked for me on iPad
Thanks

**Comment by craiglpeters** [comment, marked as answer]
These kinds of connection issues are classically hard to solve as they are so specific to each local environment (OS, client/browser, network, etc). I suggest working though the troubleshooting steps:

- [Connection Troubleshooting](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces)
- [Client Troubleshooting](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-github-codespaces-clients)

**Reply by CZNetworks** [reply]
0fee1cf514be58e524f4d159a46002870b96bed7

**Reply by alexandercarlis2-dotcom** [reply]
Akash1134

**Reply by alexandercarlis2-dotcom** [reply]
Akash1134

**Comment by abisarwan-go** [comment]
I had this problem with safari then I tried using chrome and it's solved !

**Reply by kirill-push** [reply]
Same problem, same solution!
I have an old MacBook Air with OS Big Sur and there is no way to update the OS and only switching to Chrome solved the problem. 
Thanks for the recommendation!

**Reply by EhogGawlic** [reply]
I'm already on chrome

**Reply by sanchalee19** [reply]
try switching your wifi networks to any other network... It worked for me

**Comment by devashish2024** [comment]
Same to me, only when I open VSCode on web. @dineshh-choudhary , you can try VSCode (not the web one), it works

**Comment by SowndSarva** [comment]
I had the same issue with chrome . I tried logging out of github and closed the browser. Again open the browser and login again and it works!.

**Comment by Sachinhooda44** [comment]
Simply close that tab and go to your github account
now open codespaces and there you will see your running codespaces
click on three dots and then click on open in vscode
hope this helps

**Reply by RishabhChoudhary172001** [reply]
it works for me

**Comment by pancharasshubham** [comment]
# CodeSpace is not working 
## Is any one having this issue right now, i am unable to connect to my codespace for last 10 days
 
## Troubleshoot Techniques I Used: 

1. Changing Browser To Firefox, Opera
2. Try Different Network
3. Clearing My Data and Cache 
4. Log out form account 
5. ipconfing /flushdns
6. Opening in Incognito Mode 
7. Opening in Vscode mode
8. Opening in Jupitar Lab

## **Network tab is showing all css and script are failed to load** 
**Any help is appreciated** 
**Thank You** 

![github_error](https://github.com/user-attachments/assets/b7cb9f22-5698-4631-b786-2055bf3afa4d)

**Reply by SandeepSinghSethi** [reply]
Found a working solution for this and it is very easy , I think it's the issue how your isp resolves domains, in the past github domains were accidentally blocked by my regional isp's (to ensure robust security even if entire critical development is compromised) , so for browser based tasks like codespaces or web code editor : simply change your browser dns config , and change it to like 1.1.1.1 or 8.8.8.8 in place of your system's default DNS resolver, and for application level fix , try to overwrite your resolve.conf to nameserver 1.1.1.1 , it will work for the current session , as after reboot , it will come back to what it was originally .
For permanent fix, search for your resolver (systemd-resolve/resolvectl/NetworkManager)  , and do appropriate fixes . Try not to change system's config if you don't know what you are tinkering with , as it might result in a disaster , for browser based codespaces just changes browser's dns resolver , and that's all , no vpn , no unwanted troubleshooting .

**Comment by mmoreinis** [comment]
In our case, I had been using last year's .devcontainer.json file to pre-load MS Live Preview and specifying an out-of-date image.  There is a "use latest" that I replaced that with to get it to work now:

{
  "name": "My Project Dev Container",
  "image": "mcr.microsoft.com/devcontainers/universal:latest",
  "customizations": {
    "vscode": {
      "extensions": [
        "ms-vscode.live-server"
      ]
    }
  }
}

**Comment by stelios-c** [comment]
I'm having this issue since yesterday. I tried both the open in browser and Jupyter Lab options, same issue.

**Reply by jamesontwowheels** [reply]
Same issues in this thread: https://github.com/orgs/community/discussions/175786

Are you UK/London based by chance?

**Reply by stelios-c** [reply]
Not London but yes UK.

**Reply by mmistroni** [reply]
Same issue, in UK. everything stopped working after 7pm

**Reply by mmistroni** [reply]
Ok a 'nearly empty' codespace started up in 20 min, and asked me for github authentication before kicking off. after i authenticated, it worked ok.

**Reply by mdmourao** [reply]
Having this issue for the last 3 days

**Reply by Erichea** [reply]
Same

**Comment by pancharasshubham** [comment]
I have had this issue for months I haven't worked on one of the projects
since then. If anyone has a solution please let me know.

**Comment by Darshab00** [comment]
if your currently faceing this issue in chrome try to change ur DNS into 8.8.8.8 in your wifi setting or change your connection network

**Reply by mdmourao** [reply]
change connection network does not solve my issue, I tried in multiple computers and networks

**Reply by Darshab00** [reply]
change the wifi DNS into 8.8.8.8 and alternatively DNS to 8.8.4.4 this will slove the issue...if the problem persist give more details like ur browser and wifi settings in ur device

**Comment by mdmourao** [comment]
Same issue for me:

https://github.com/orgs/community/discussions/175866

Also found:

https://github.com/Microsoft/vscode/issues/268971

**Comment by KeiranGamingTV** [comment]
If you're on an organization-managed device, such as a school or work laptop, your organization may have blocked [https://main.vscode-cdn.net](https://main.vscode-cdn.net/). This runs the stylesheet, preconnect, dns-prefetch, and script for [vscode.dev](https://vscode.dev), which is what Github Codespaces runs on. Try opening vscode.dev, and if it does not load, try opening main.vscode-cdn.net. If that website is blocked, that could be the root of your problem.
UPDATE: After running it through a site checker, it shows that [main.vscode-cdn.net](https://main.vscode-cdn.net) is currently down, so it could be something on Microsoft's side. Might be related to the recent Microsoft Azure outage.

**Comment by mmistroni** [comment]
happened to me recently  after a month... i simply try to run on Us-EAst,
which seems more reliable
hth

**Reply by KeiranGamingTV** [reply]
Didn't seem to work for me. My organization has [main.vscode-cdn.net](https://main.vscode-cdn.net/) blocked completely, so vscode.dev won't even load entirely.
Also, running it through a site checker shows that the site is down, so it could be something on Microsoft's side. Might be related to the recent Azure outage.

**Comment by praveenkumarlodhi979-tech** [comment]
Ham to Abhi Gyan Arjit kar rahe hain Kuchh samay mein Sikh sakta hun mujhe aisa lagta hai aur safalta prapt kar sakta hun is karya ke liye Maine yah ho sakta hai mujhe Bharosa Hai

**Reply by wifi-beater** [reply]
nice

**Comment by sholt6** [comment]
In case someone else comes across this and finds it helpful, I had this problem with a Codespace which I was sure had worked in the past, and it worked again only once I turned off my company VPN.

**Comment by irfanamahboob** [comment]
I am facing this today. I thought its is because of office latop - but i am stuk in the same page in personal laptop also. Changed networks , changed browsers, - nothing works - can anyone pls help?

<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/99bb1e68-0374-4401-99d6-9237fe255b7d" />

**Comment by mmistroni** [comment]
Try new zone..or check updates ..last time it happened to me there were
general problems on GitHub code spaced
Hth

**Comment by CyberNFT** [comment]
👍🏻
